doit/doit_cmd.py

```diff
@@ -49,7 +49,7 @@ def set_var(name, value):
 class DoitConfig():
     """Parse and store values taken from INI and TOML configuration files"""
     # support TOML python libs
-    _TOML_LIBS = ['toml', 'tomlkit']
+    _TOML_LIBS = ['toml', 'tomli', 'tomlkit']
```

Comment:
perhaps it would make sense to check `tomli` first, and move `toml` to the end, as `tomlkit` is at least still being maintained.

Further, it could start looking for [PEP 680 `tomllib`](https://peps.python.org/pep-0680/).

Reply:
Sure, will do that.